decidim-verifications/lib/decidim/verifications/hooks.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Verifications
+    class Hooks
```

Comment:
This should probably be called `DefaultHooks`, as it comes with a default implementation.

Reply:
Also, not sure about `hook` as a name for this abstraction.

Reply:
What about calling it `DefaultAuthorizationHandlerAuthorizer`? it is a little long, but I think is the most accurate name, since it provides the logic for `ActionAuthorizer` that is inside the specific authorization handler. I would also change all references to `hooks` with `authorizer`.

Reply:
What about using `DefaultActionAuthorizer` and `action_authorizer` as a method name?

Reply:
Ok, I will prepare that change. :+1: